packages/playwright-test/src/types.ts

```diff
@@ -40,6 +40,7 @@ export interface TestStepInternal {
  * increasing the surface area of the public API type called FullConfig.
  */
 export interface FullConfigInternal extends FullConfigPublic {
+  globalOutputDir: string;
```

Comment:
Everything here should be underscored, to not confuse folks printing their full configs. Might as well add a comment for this.

Reply:
Agreed—when this PR was first authored, there were no other members here! But now that it has friends, definitely should be underscored!